chart/env/prod.yaml

```diff
@@ -389,6 +390,12 @@ search:
   replicas: 7
   service:
     type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      alb.ingress.kubernetes.io/group.order: "1"
+      alb.ingress.kubernetes.io/target-node-labels: role-datasets-server-search=true
+      alb.ingress.kubernetes.io/target-group-attributes: stickiness.enabled=true,stickiness.lb_cookie.duration_seconds=300
```

Comment:
ok, but I don't think we pass cookies anyway

Reply:
let's try ;) 
The LB should create a cookie during the first call to remember the instance. During the 2nd call, the ALB will read the cookie, and redirect to the same instance.

Reply:
Ah ok!